ludwig/features/base_feature.py

```diff
@@ -103,6 +103,14 @@ def add_feature_data(
         raise NotImplementedError
 
 
+class PredictModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.predictions_key = PREDICTIONS
```

Comment:
What's the reasoning behind using `self.predictions_key` instead of referencing constants directly?

It looks like all this base class is doing is storing common keys. If we instead reference global constants directly, then perhaps this PredictModule isn't needed at all, or is this required by torchscript?

Reply:
Without setting these constants to properties, Torchscript will complain during scripting. It seems scripting is unable to pull in global variables.

Reply:
Will add a comment.